templates/contact-us.html

```diff
@@ -5,39 +5,131 @@
     <div class="p-strip--suru-image">
       <div class="row">
         <div class="col-6">
-          <img src="https://assets.ubuntu.com/v1/6685f10c-contact-canonical-icon.svg" alt="Canonical contact icon">
+          <h1 class="u-off-screen">Contact Canonical</h1>
+          <img src="https://assets.ubuntu.com/v1/6685f10c-contact-canonical-icon.svg" alt="Contact Canonical">
```

Comment:
There's no need for the alt value, as the  adjacent `h1` conveys the same information (see [this PR](https://github.com/canonical-web-and-design/ubuntu.com/pull/6109#issue-339863239) from mpt, where he does into depth on the subject).

```suggestion
          <img src="https://assets.ubuntu.com/v1/6685f10c-contact-canonical-icon.svg" alt="">
```